### PR TITLE
fix(doc): Correct readme typos

### PR DESCRIPTION
**Pull request by cyrilf**
For the respect of the Baguette! :fr: :baguette_bread: 

Merci. :sparkles:

### CODE REVIEW

File: lib/ExtensionPoint/Readme.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Render default content when no extension found
 
 Render an extension
 
-<small>Try to change the `extensionName` to `Hello1` to see the default rendering provided by the Extension Point.</small>
+<small>Try to change the `extensionName` to `Hello` to see the default rendering provided by the Extension Point.</small>
```
Comment (pke) [inline lib/ExtensionPoint/Readme.md]: Changing it to `Hello` will not show the default rendering, since the Extension is named "Hello". Try it out :)

**Review by pke — CHANGES_REQUESTED** [review]

**Comment by pke** [comment]
Thank you for your PR. Can you please rebase and resubmit?

**Comment by cyrilf** [comment]
Sure.
Rebased!

**Review by pke — APPROVED** [review]

File: lib/ExtensionPoint/Readme.md (hunk 1 above)
Comment (cyrilf) [reply lib/ExtensionPoint/Readme.md]: I got it now. I though it was describing the code below.  
I forgot that we can directly play with the code from the doc page, now it makes sense. :)

**Pull request merged**